english/bite-the-marshmallow/tasty-marshmallow.md

```diff
@@ -15,4 +15,6 @@ Wait a minute....how did you end up in this car?  You must be hallucinating
 
 [You open the door and there is a man sitting on your couch](../movie-ripoffs/zoolander/zoolander.md)
 
-[Ask the driver "Where are we going?!"](ricky/ricky.md)
\ No newline at end of file
+[Ask the driver "Where are we going?!"](ricky/ricky.md)
+
+[Procrastinate with social media](lamping/lamping.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.